notebooks/Add GPS Metadata Fields.ipynb

```diff
@@ -11,26 +11,37 @@
     "- Use this script to update an existing line or polygon feature class to support GPS metadata\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.        #if arcpy.Describe(feature_class).shapeType != "Polyline" and arcpy.Describe(feature_class).shapeType != "Polygon":
</pre><p>Is this supposed to be commented out?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/field-maps-scripts/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Bah, thanks for catching that. Updated